wemake_python_styleguide/constants.py

```diff
@@ -369,6 +369,9 @@
 #: Maximum length of ``yield`` ``tuple`` expressions.
 MAX_LEN_YIELD_TUPLE: Final = 5
 
+#: Maximum length of ``return`` ``tuple`` expressions.
+MAX_LEN_RETURN_TUPLE: Final = 5
```

Comment:
I think that we should unify this setting into a single one. How about `MAX_LEN_TUPLE_OUTPUT`?

Reply:
Okay. Should I rename `ReturnLikeStatementTupleVisitor` too?